src/utils/__tests__/transformToHarRequest.test.js

```diff
@@ -0,0 +1,75 @@
+import transformToHarRequest from '../transformToHarRequest';
+
+describe('transformToHarRequest', () => {
+  it('transforms GET request', () => {
+    const data = {
+      method: 'GET',
```

Comment:
@MeganSteenkamp We have `PUT` and `DELETE` APIs too :)

Reply:
@tom-blake do you think those need testing?
Maybe I should change the description of the tests as I think GET and POST cover all scenarios - i.e. post data and no post data.

Reply:
@MeganSteenkamp Yep making the test generic is fine

Suggestion: Rename `postData` to `payload`. It's a more general name 👍